src/vm/readytoruninfo.h

```diff
@@ -40,6 +41,9 @@ class ReadyToRunInfo
     Crst                            m_Crst;
     PtrHashMap                      m_entryPointToMethodDescMap;
 
+    PersistentInlineTrackingMapR2R  m_persistentInlineTrackingMap;
+    PTR_PersistentInlineTrackingMapR2R m_pPersistentInlineTrackingMap;
```

Comment:
Mostly I just did it because the machinery to make new allocations looks like it requires some additional arguments to be passed around and the map datastructure is only a few pointers in size, however if I am breaking form I can certainly do a seperate allocation.